kanidmd/src/lib/interval.rs

```diff
@@ -24,4 +26,14 @@ impl IntervalActor {
             }
         });
     }
+
+    pub fn start_live_backup(server: &'static QueryServerReadV1) {
+        tokio::spawn(async move {
+            let mut inter = interval(Duration::from_secs(5));
```

Comment:
Rather than a duration, this should be triggered at a configured time in the server.toml I think. IE 10pm daily or hourly or similar.

Reply:
cron calendar support, perhaps? :) [saffron](https://docs.rs/saffron/0.1.0/saffron/parse/index.html) is what CloudFlare made to parse things.

Reply:
I was also thinking of a cron like thing, but then thought it might be too much, too complicated?

Thanks for the hint, will look into it.

It probably also adds a lot of flexibility.

How should it be? We could also start simple and extend. 😉